api/src/main/java/io/minio/MinioClient.java

```diff
@@ -2499,8 +2499,33 @@ public void copyObject(CopyObjectArgs args)
             null,
             null,
             0)) {
-      // For now ignore the copyObjectResult, just read and parse it.
-      Xml.unmarshal(CopyObjectResult.class, response.body().charStream());
+
+      String bodyContent = new String(response.body().bytes(), StandardCharsets.UTF_8);
```

Comment:
Why can't it be just ` Xml.unmarshal(CopyObjectResult.class, response.body().charStream());` ? There is no need to read the body to a string and parse it later.